blocks/libs/test/qa_KnownSharedLibBlocks.cpp

```diff
@@ -0,0 +1,92 @@
+#include <boost/ut.hpp>
+
+#include <gnuradio-4.0/Block.hpp>
+#include <gnuradio-4.0/BlockRegistry.hpp>
+
+#include <iostream>
+
+using namespace boost::ut;
+
+using namespace std::string_view_literals;
+
+extern "C" {
+bool gr_blocklib_init_module_GrFilterBlocks(gr::BlockRegistry& registry);
+bool gr_blocklib_init_module_GrFourierBlocks(gr::BlockRegistry& registry);
+bool gr_blocklib_init_module_GrElectricalBlocks(gr::BlockRegistry& registry);
+bool gr_blocklib_init_module_GrHttpBlocks(gr::BlockRegistry& registry);
+bool gr_blocklib_init_module_GrFileIoBlocks(gr::BlockRegistry& registry);
+bool gr_blocklib_init_module_GrTestingBlocks(gr::BlockRegistry& registry);
+bool gr_blocklib_init_module_GrBasicBlocks(gr::BlockRegistry& registry);
+}
```

Comment:
Is this needed just for this unit-tests or expected for all user programs?

Reply:
Thanks ... yes, would be better if the block-lib user/writer doesn't need to know/care about this.